apollo-router/src/plugins/demand_control/mod.rs

```diff
@@ -209,6 +210,20 @@ pub(crate) struct DemandControl {
     strategy_factory: StrategyFactory,
 }
 
+impl DemandControl {
+    fn report_operation_metric(context: Context) {
+        let guard = context.extensions().lock();
+        let cost_context = guard.get::<CostContext>();
+        let result = cost_context.map_or("NO_CONTEXT", |c| c.result);
+        u64_counter!(
+            "apollo.router.demand_control",
```

Comment:
```suggestion
            "apollo.router.operations.demand_control",
```

Reply:
I added explicit guidance here: https://github.com/apollographql/router/blob/15364ebb3916e96c07be98aa5a22fae3d9e906f8/dev-docs/metrics.md#static-metrics

Reply:
Updated in https://github.com/apollographql/router/pull/5114/commits/bd6f26860ac3c4772ef1450cb2b8ad3e40963f20